vancouver-imperial-college-london.csl

```diff
@@ -77,10 +85,9 @@
     <choose>
       <if variable="URL DOI" match="any">
         <group delimiter=" ">
-          <text term="available at" text-case="capitalize-first" suffix=":"/>
           <choose>
             <if variable="DOI">
-              <text variable="DOI" prefix="doi:"/>
+              <text variable="DOI" prefix="doi:" suffix="."/>
```

Comment:
shouldn't the prefix be "https://doi.org" in line with the guidelines?

Reply:
No, this should stay just as ‘doi’, we don’t include the https: part

Reply:
FWIW, your guidance is contradictory here. You have some examples with doi: but also 
![Screenshot_20211106-153353.png](https://user-images.githubusercontent.com/128986/140621722-f8de5c9e-850b-42f4-9b64-b2f0f551b97a.png)

And doi: is discouraged by CrossRef and the DOI foundation. Any chance you could change/unify that in the guidance?

Reply:
changed o https/..... as requested